### DISCUSSION TITLE
Is it possible to add plugin support?

### DISCUSSION THREAD

**Discussion by darwintree** [audioling/audioling#1] [Ideas]
There are cases when I would like to add additional features for feishin, for example, fetching lyrics or song/artist information from a private server and display.
Would it be possible that audioling can provide plugin support for users to customize the function of the music player?

**Comment by jeffvli** [comment]
This is something I'll consider, but I would have to rearchitect the entire app is it stands right now to accomplish that.

I'll do some research in the next few days to see how feasible it would be.
